api/test/deals.go

```diff
@@ -1,26 +1,33 @@
 package test
 
 import (
+	"bytes"
 	"context"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"math/rand"
 	"os"
+	"path/filepath"
 	"testing"
 	"time"
 
 	logging "github.com/ipfs/go-log"
 
 	"github.com/filecoin-project/lotus/api"
-	"github.com/filecoin-project/lotus/chain/address"
+	"github.com/filecoin-project/lotus/build"
 	"github.com/filecoin-project/lotus/chain/types"
 	"github.com/filecoin-project/lotus/node/impl"
 )
 
+func init() {
+	logging.SetAllLoggers(logging.LevelInfo)
+	build.InsecurePoStValidation = true
```

Comment:
Debug stuff.

Reply:
That's a test package

Reply:
I've seen test packages get included into main builds.

Reply:
This prints a ton of spam, and makes it impossible to sync correctly, so if that happens, it's impossible to miss.

If you know of a simple/relaiable way to detect running in a go test, then we can use that, but I'm going to say it's not worth the effort to fix now